databuilder/models/table_metadata.py

```diff
@@ -66,6 +66,7 @@ class TableMetadata(Neo4jCsvSerializable):
     TABLE_NODE_LABEL = 'Table'
     TABLE_KEY_FORMAT = '{db}://{cluster}.{schema}/{tbl}'
     TABLE_NAME = 'name'
+    IS_VIEW = 'is_view'
```

Comment:
We may want to update the comment about the class to indicate it could be used for both table and view.